frontend/public/components/utils/details-page.tsx

```diff
@@ -65,11 +65,19 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
     namespace: metadata.namespace,
   });
 
+  let label = 'Namespace';
+  if (kind === 'Subscription') {
+    label = 'Operator Namespace';
+  }
+  if (kind === 'ClusterServiceVersion') {
+    label = 'Managed Namespace';
+  }
```

Comment:
I agree with @christianvogt that this component should be generic and not know about these kinds.

From a UX standpoint, I also think this change is confusing as well. Calling it something other than `Namespace` makes me think the resource is in a different namespace and is just managing this namespace, which is not the case. It makes me think I can change the managed namespace, which I can't.

This component is here to ensure consistency across pages, and every other page calls this `Namespace`. `Namespace` also matches what the CLI and YAML show. It's hard to change this label because it *should* be hard to change this label. Changing it creates an inconsistency.

Reply:
@alimobrem @tlwu2013 @dmesser what are your thoughts on this? Curious to hear what you think, as well.

Reply:
I see the value in indicating to the user what the actual labels are so that they can map them to the CLI/YAML, and also pulling the contents of the popover from API descriptions to keep them current. I do think though, that doesn't leave much room in these details screens to inject any additional UX niceness (further describing a namespace as "operator" or "managed" for example.)

Perhaps we could have some common paradigm where these labels are allowed to be changed in the UI (some sort of 'display name') and if that is present, add the actual resource label name is added to the popover contents for reference.

"Appears as `Namespace` in the `ClusterServiceVersion` YAML."

Alternatively, maybe these popovers could be divided into two sections, an optional 'ui-specified' description that appears first, followed by the API description.

It would seem unfortunate to try to further describe these two kinds of namespaces in the list view's column headers and not have any indication of which are which in the resource details.